cranelift-entity/src/set.rs

```diff
@@ -93,6 +103,21 @@ where
         self.elems[index / 8] |= 1 << (index % 8);
         result
     }
+
+    /// Removes and returns the entity from the set if it exists.
+    pub fn pop(&mut self) -> bool {
```

Comment:
Suppose entries 5 and 7 exist. The len will be 8 at first. When calling pop, entry 7 is removed and the new len is 7. When calling pop again, entry 6 is examined. Because it is not existing pop returns false, even though there is an existing entry.